src/controllers/vdeployment_controller/trusted/liveness_theorem.rs

```diff
@@ -20,20 +20,36 @@ pub open spec fn vd_eventually_stable_reconciliation_per_cr(vd: VDeploymentView)
 // TODO: add another version which talks about pods and derives from VRS ESR and this ESR
 pub open spec fn current_state_matches(vd: VDeploymentView) -> StatePred<ClusterState> {
     |s: ClusterState| {
-        let objs = s.resources().values().filter(|obj: DynamicObjectView| {
-            &&& obj.kind == VReplicaSetView::kind()
-            &&& obj.metadata.namespace == vd.metadata.namespace
-        }).to_seq();
-        let vrs_list = objects_to_vrs_list(objs);
-        let filtered_vrs_list = filter_vrs_list(vd, vrs_list.unwrap());
-        let (new_vrs, old_vrs_list) = filter_old_and_new_vrs(vd, filtered_vrs_list);
-        &&& vrs_list.is_Some()
+        // simulate handle_list_req
```

Comment:
Is this change for making the spec consistent with API server's state machine so the proof will be easier?